src/components/Banner/banner.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './banner.module.scss';
+
+export function Banner() {
+    const [displayBanner, setDisplayBanner] = React.useState(true);
+
+    const display = displayBanner ? styles.banner : styles.hidden;
+
+    function toggleBanner() {
+        setDisplayBanner((prevState) => !prevState);
+    }
```

Comment:
If I'm reading this right, this banner will come back as soon as someone refreshes the page.

Could we update this toggling code to 
- When the state changes store that value in local storage or a cookie?
- On initial render, read that value from local storage or the cookie to properly set the initial state?


It's worth noting that if you do this, we'll likely need to not render any content until the component is fully mounted as during initial render (which typically happens server side) that initial state we need to read from won't be there.

Reply:
I tried to implement this as requested but I was unable to. I returned it to the version of the code you have noted, and yes, the component does function like you stated.

Reply:
Thanks to Kayla, I was able to get the requested change functioning! This PR should be good to merge!